_pages/index.md

```diff
@@ -13,20 +13,54 @@ layout: home
   </div>
 </section>
 
-<section class="usa-section grid-container usa-prose" markdown="1">
+<section class="usa-section grid-container" markdown="1">
 
   {% if site.temporary_alert %}
     {% include alert.html content=site.temporary_alert %}
   {% endif %}
 
-# Get started...
+<h2 class='margin-top-0'>How to integrate with Login.gov</h2>
 
-- [Understand our flow]({{ site.baseurl }}/overview/).
-- Determine your application needs, like the level of proofing and [user attributes]({{ site.baseurl }}/attributes/) that will be requested.
-- Select between [OpenID Connect (OIDC)]({{ site.baseurl }}/oidc/) or [SAML protocol]({{ site.baseurl }}/saml/) implementation protocols. Please note that we recommend OIDC.
-- Configure your app. We have [implementation guides]({{ site.baseurl }}/oidc/) and example apps to get you up and running quickly.
-- [Register your app in the sandbox dashboard and start testing]({{ site.baseurl }}/testing/).
-- Let us know when you are ready to go live and our team will help you [promote the application to production]({% link _pages/production.md %}). We will check you against our production checklist to ensure your application is good to go to production from an administrative and technical standpoint.
-- Be sure to read the [FAQ]({{ site.baseurl }}/support/#frequently-asked-questions) for answers to the most common questions.
+<ol class="usa-process-list usa-prose margin-bottom-4">
+  <li class="usa-process-list__item">
+    <p>
+      Learn about the process the user will go through to access your application and what steps are involved in configuring your app on the <a href="{% link _pages/overview.md %}" class="usa-link">Integration</a> page.
+    </p>
+  </li>
 
+  <li class="usa-process-list__item">
+    <p>
+      Determine your application needs, like the level of proofing, the <a href="{% link _pages/design-guidelines.md %}" class="usa-link">user experience</a> decisions your agency will make, and the <a href="{% link _pages/attributes.md %}" class="usa-link">user attributes</a> that will be requested.
+    </p>
+  </li>
+
+  <li class="usa-process-list__item">
+    <p>
+      Select between <a href="{% link _pages/oidc.md %}" class="usa-link">Open ID Connect</a> (OIDC) or <a href="{% link _pages/saml.md %}" class="usa-link">SAML</a> implementation protocols.
+    </p>
+  </li>
+
+  <li class="usa-process-list__item">
+    <p>
+      Configure your app in the <a href="https://dashboard.int.identitysandbox.gov/" class="usa-link">sandbox dashboard</a> and start <a href="{% link _pages/testing.md %}" target="_blank" class="usa-link">testing</a>! We also have examples of <a href="https://github.com/18F/identity-oidc-sinatra" class="usa-link">OIDC</a> and <a href="https://github.com/18F/identity-saml-sinatra" target="_blank" class="usa-link">SAML</a> apps to get you up and running quickly.
```

Comment:
Only one of the two sample links here use `target="_blank"`. Should they both? Also, `target="_blank"` is added for the "Testing" link, but that's internal to the site.

We could also add the external link icon here if it's going to a different domain / opening in a new tab.

```suggestion
      Configure your app in the <a href="https://dashboard.int.identitysandbox.gov/" target="_blank" class="usa-link usa-link--external">sandbox dashboard</a> and start <a href="{% link _pages/testing.md %}" class="usa-link">testing</a>! We also have examples of <a href="https://github.com/18F/identity-oidc-sinatra" target="_blank" class="usa-link usa-link--external">OIDC</a> and <a href="https://github.com/18F/identity-saml-sinatra" target="_blank" class="usa-link usa-link--external">SAML</a> apps to get you up and running quickly.
```